client/src/main/java/com/paypal/selion/internal/platform/grid/browsercapabilities/SelendroidCapabilitiesBuilder.java

```diff
@@ -35,30 +35,31 @@
     @Override
     public DesiredCapabilities getCapabilities(DesiredCapabilities capabilities) {
 
+        DesiredCapabilities tempCapabilities = capabilities;
```

Comment:
Here the _Object reference_  to `capabilities` is copied. So whether someone uses `capabilities` and `tempCapabilities`, refer to the same object.

Thus, this change is not really fixing the `Avoid reassigning parameters such as 'capabilities'` condition.

There are several places in the PR where this issue occurs.  

In this particular case, the approach should be 
```java
DesiredCapabilties newCapabilities = SelendroidCapabilities.android(); // Note: SelendroidCapabilities.android() returns a new DesiredCapabilties object. So,now that the original `capabilities` _Object reference_ and values are lost.
newCapabilities.merge(capabilities); // I suspect the fact that we don't merge the pre-existing values is likely a bug we have today and might need to fix here and in other places.

// the rest of the changes
```